stratos/chain.json

```diff
@@ -112,6 +120,9 @@
       {
         "address": "stratos.grpc.nodersteam.com:9090",
         "provider": "[NODERS]TEAM"
+      {
```

Comment:
Close the set above this. Are you also running two grpc end points or was this intended to be a replacement?

Reply:
also fixed :), these are two different nodes, maybe we will remove the one above later